cluster/addons/device-plugins/nvidia-gpu/daemonset.yaml

```diff
@@ -36,7 +36,7 @@ spec:
         hostPath:
           path: /dev
       containers:
-      - image: "gcr.io/google-containers/nvidia-gpu-device-plugin@sha256:0e79da6998a61257585e0d3fb5848240129f0fa5b4ad972dfed4049448093c33"
+      - image: "gcr.io/google-containers/nvidia-gpu-device-plugin:31ee590ea32667804438a380cea5f46064cf9f03"
```

Comment:
This is already at the latest version. The sha corresponds to https://github.com/GoogleCloudPlatform/container-engine-accelerators/commit/8a23fdc9d635fee1a91b1f9947bb80a7a0edaa5c

There's no difference in the [device plugin code](https://github.com/GoogleCloudPlatform/container-engine-accelerators/blob/master/cmd/nvidia_gpu/nvidia_gpu.go) between that commit and the latest commit: https://github.com/GoogleCloudPlatform/container-engine-accelerators/compare/8a23fdc9d635fee1a91b1f9947bb80a7a0edaa5c...31ee590ea32667804438a380cea5f46064cf9f03

I would prefer to keep the existing sha to avoid unnecessary daemonset updates on already running clusters.